binder/src/androidTest/java/io/grpc/binder/internal/LeakSafeOneWayBinderTest.java

```diff
@@ -71,7 +78,21 @@ public void testDetach() {
     assertThat(binder.onTransact(456, p, null, FLAG_ONEWAY)).isFalse();
 
     // The transaction shouldn't have been processed.
-    assertThat(transactionsHandled).isEqualTo(0);
+    assertThat(handler.transactionsHandled).isEqualTo(0);
+
+    p.recycle();
+  }
+
+  @Test
+  public void testReplace() {
+    binder = new LeakSafeOneWayBinder(handler);
```

Comment:
The field is already initialized this way. Is this no-op?

Reply:
Yes but "Use setup methods to define implicit defaults for tests, but never let tests explicitly rely on values defined in setup methods" (go/unit-testing-practices#setUp)

This test case needs to explicitly show the reader the relationship between `binder` and `handler` without relying on them to scroll way up and read the body of the setup method.